pkgs/by-name/pl/plattenalbum/package.nix

```diff
@@ -1,25 +1,25 @@
 { lib, fetchFromGitHub
-, pkg-config, meson ,ninja
+, pkg-config, meson, ninja
 , python3Packages
-, gdk-pixbuf, glib, gobject-introspection, gtk3
+, gdk-pixbuf, glib, gobject-introspection, gtk4, libadwaita, desktop-file-utils
 , libnotify
 , wrapGAppsHook }:
 
 python3Packages.buildPythonApplication rec {
-  pname = "mpdevil";
-  version = "1.11.0";
+  pname = "plattenalbum";
+  version = "2.0.1";
 
   src = fetchFromGitHub {
     owner = "SoongNoonien";
     repo = pname;
     rev = "refs/tags/v${version}";
-    sha256 = "sha256-ooNZSsVtIeueqgj9hR9OZp08qm8gGokiq8IU3U/ZV5w=";
+    sha256 = "sha256-rxHtcKbvwoOteSD3gC0uEp+3GCUFpKWoicznELVilpY=";
   };
 
   format = "other";
 
   nativeBuildInputs = [
```

Comment:
It's better to make this list vertical, but not a big problem since the official formatter will be out soon